api/managementpb/backup/backups.proto

```diff
@@ -0,0 +1,45 @@
+syntax = "proto3";
+
+package backup.v1beta1;
+
+option go_package = "api/managementpb/backup;backupv1beta1";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+
+message RestoreBackupRequest {
+  // Service identifier where backup should be restored.
+  string service_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Machine-readable location ID.
+  string location_id = 2 [
```

Comment:
this seems redudandant, you can retrieve location from artifact

Reply:
Hm, you right. @oter can we eliminate `location_id` here?

Reply:
yes, removing it.